hurumap/urls.py

```diff
@@ -5,4 +5,4 @@
 from hurumap import settings
 
 urlpatterns = static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + \
-              wazimap_urlpatterns
+              list(wazimap_urlpatterns)
```

Comment:
`static` returns a list of urls and `wazimap_urlpatterns` is also a list of urls (if upgraded to Django 1.11 properly) so this should work without an explicit cast to list. If it doesn't work, then you've uncovered a bug in `OpenUpSA/wazimap/releases`

Reply:
@kilemensi `urlpatterns` in wazimap are a tuple. That was the reason for the explicit cast to list

Reply:
@phillipahereza 

From Django 1.8, the use of `turple`/`pattern()` was removed and `urlpatterns` are not expected to be a list: https://docs.djangoproject.com/en/2.1/releases/1.8/#django-conf-urls-patterns

Checkout `wazimap/master`, it's been upgraded to Django 1.11 as well and there, the `urlpatterns` were converted to a list... So if this explicit cast works, lets go ahead with it **but** this means something was left out in the `OpenUpSA/wazimap/releases` upgrade.

Reply:
@kilemensi I've updated our releases branch and opened a PR with the upstream to fix above issue